pkgs/applications/version-management/bcompare/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, qt511, libsForQt511, cairo, gdk_pixbuf, gnome2, bzip2, qt4, lib, fetchurl, autoPatchelfHook, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "bcompare";
+  version = "4.2.10.23938";
+
+  src = fetchurl {
+    url = "https://www.scootersoftware.com/${pname}-${version}_amd64.deb";
+    sha256 = "5471d2bcb9494830bc706a940c8a598ecf7a1e56aa7e26c2694048b568d045a0";
+  };
+
+  unpackPhase = ''
+    ar x $src
+    tar xfz data.tar.gz
+  '';
+
+  installPhase = with stdenv.lib; ''
+    mkdir -p $out/bin $out/lib $out/share
+    cp -R usr/share $out/
+    cp -R usr/lib $out/
+    cp -R usr/bin $out/
+
+    # Remove library that refuses to be autoPatchelf'ed
+    rm $out/lib/beyondcompare/ext/bcompare_ext_kde.amd64.so
+
+    substituteInPlace $out/bin/bcompare \
+      --replace "/usr/lib/beyondcompare" "$out/lib/beyondcompare" \
+      --replace "/bin/bash" "/usr/bin/env bash"
+
+    # Create symlink bzip2 library
+    ln -s ${bzip2.out}/lib/libbz2.so.1 $out/lib/beyondcompare/libbz2.so.1.0
+
+  '';
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    gnome2.gtk
```

Comment:
```suggestion
    gtk2
```

Reply:
I get 
`$ nix-build -A bcompare
error: undefined variable 'gtk2' at /home/ktor/development/projects/nixpkgs/pkgs/applications/version-management/bcompare/default.nix:39:5`
when I change that.

Reply:
Did you add it to the function arguments at the top of the  file?

Reply:
fixed :)